docs/.eslintrc.json

```diff
@@ -1,3 +1,18 @@
 {
-  "extends": "next/core-web-vitals"
+  "plugins": ["prettier"],
+  "extends": ["next/core-web-vitals", "plugin:mdx/recommended", "prettier"],
+  "settings": { "mdx/code-blocks": true },
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "overrides": [
+    {
+      "files": ["*.angular.mdx"],
```

Comment:
override Angular files because the code block would have the following error

> Support for the experimental syntax 'decorators-legacy' isn't currently enabled (6:1):